ui/src/testing/stubs/workspaces-api-stub.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  WorkspacesApi
+} from 'generated/fetch';
+
+export class WorkspacesApiStub extends WorkspacesApi {
```

Comment:
why do we need this?

Reply:
Mounting the component in tests calls the `componentDidMount` lifecycle hook, which tries to make an ajax request, so we need to stub it out.